website/docs/docs/build/incremental-microbatch.md

```diff
@@ -280,6 +282,123 @@ For now, dbt assumes that all values supplied are in UTC:
 
 While we may consider adding support for custom time zones in the future, we also believe that defining these values in UTC makes everyone's lives easier.
 
+## Parallel batch execution
+
+The microbatch strategy offers the benefit of updating a model in smaller, more manageable batches. 
+
+Parallel batch execution means that multiple batches are processed at the same time, instead of one after the other (sequentially) for faster processing of your microbatch models. 
+
+For example, if you have a microbatch model with 12 batches, you can execute those batches to run in parallel. Specifically they'll run in parallel limited by the number of [available threads](/docs/running-a-dbt-project/using-threads).
+
+### Prerequisites
+
+To enable parallel execution, you must meet the following conditions:
+
+- You use the following supported adapters:
+  - Snowflake
+  - Databricks
+  - More adapters coming soon!
+- The relation in the data warehouse for the model doesn't exist
```

Comment:
@nataliefiann confirming that relation is the same as {{ this }}

Reply:
Hiya @QMalcolm 

I just wanted to ask about this sentence: "The relation in the data warehouse for the model doesn't exist"
Does the relation refer to {{ this }} ?

Kind Regards
Natalie

Reply:
To answer the question, yes and no? `{{ this }}` is a python [Relation](https://docs.getdbt.com/reference/dbt-classes#relation) object for the current model. A `Relation` object is essentially just a pointer to where something should be in the data warehouse. However, having a `Relation` object doesn't necessarily mean that something exists in the data warehouse at the path that the `Relation` points to

So another way of thinking about
> The relation in the data warehouse for the model doesn't exist is

is 
> No model exists in the data warehouse yet for the path specified by the relation